api/errors/errors.go

```diff
@@ -29,3 +33,35 @@ func NewErrGasPriceTooLow(gasPrice *big.Int) *ErrGasPriceTooLow {
 		GasPrice: gasPrice,
 	}
 }
+
```

Comment:
Let's move these errors into `models/errors`, these API errors should only be API specific things, like invalid request. I feel these would be better to have under models errors as they are more domain errors.